Server/server.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+const app = express();
+const Image = require('../DB/image.js')
+const bodyParser = require('body-parser')
+app.use(express.json());
+const path = require('path');
+const babelPolyFill = require('@babel/polyfill');
+
+app.use(express.static(path.join(__dirname, '/../dist')));
+
+app.get('/', (req, res) => {
+  res.end('Baby Steps!')
+})
+
+app.get('/api/products', function(req, res) {
+  Image.find({}, function(err, result) {
+    if (err) {
+    throw err;
```

Comment:
throwing an error will cause express to return 500 error to client. Is this what you want here? Just checking.
https://expressjs.com/en/guide/error-handling.html#:~:text=Errors%20that%20occur%20in%20synchronous,will%20catch%20and%20process%20it.&text=If%20there%20is%20no%20error,catches%20and%20processes%20the%20error.